### PR TITLE
feat(rest-connector): Update docs for Rest-http Connector

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## What is the purpose of the change

Update [documentation](https://docs.camunda.io/docs/next/components/integration-framework/connectors/out-of-the-box-connectors/rest/) so it reflects the new element template.

## Are there related marketing activities

https://github.com/camunda/team-connectors/issues/138

## When should this change go live?

-

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by christinaausley — DISMISSED** [review]
Nice updates 👍  I made some minor adjustments to the connection timeout section in each of the versions in my latest commit.

**Comment by igpetrov** [comment]
@MoAtCamunda could you please review this doc?

File: docs/components/connectors/out-of-the-box-connectors/rest.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ To use a **REST Connector** in your process, either change the type of existing
 
 ## Make your REST Connector executable
 
-To make the **REST Connector** executable, fill out the mandatory fields highlighted in red in the properties panel:
+To make the **REST Connector** executable, choose needed authentication type and fill out the mandatory fields highlighted in red in the properties panel:
```
Comment (MoAtCamunda) [inline docs/components/connectors/out-of-the-box-connectors/rest.md:15]:
```suggestion
To make the **REST Connector** executable, choose the required authentication type and fill out the mandatory fields highlighted in red in the properties panel:
```

**Review by MoAtCamunda — CHANGES_REQUESTED** [review]

**Comment by akeller** [comment]
This looks like it's ready to merge after incorporating @MoAtCamunda's change. @Oleksiivanov, would you agree?

**Comment by igpetrov** [comment]
@akeller let's wait for some time. QAs are testing a related feature (timeout) at the moment.

**Review by igpetrov — APPROVED** [review]

**Comment by akeller** [comment]
@igpetrov I'll add the `hold` label. Please remove it when you are ready to merge.

**Comment by igpetrov** [comment]
> @igpetrov I'll add the `hold` label. Please remove it when you are ready to merge.

Perfect, thank you!

@Oleksiivanov FYI

**Review by christinaausley — DISMISSED** [review]

File: docs/components/connectors/out-of-the-box-connectors/rest.md [hunk 2]
```diff
@@ -41,8 +42,9 @@ We advise you to keep your **Password** safe and avoid exposing it in the BPMN `
 
 Select the **REST Connector** and fill out the following properties under the **Authentication** section:
 
-1. Set **Username** (i.e. `YOUR_USERNAME`).
-2. Set **Password** to the secret you created (i.e. `secrets.REST_BASIC_AUTH_SECRET`).
+1. Click **Basic Auth** in the **Authentication** section.
+2. Set **Username** (i.e. `YOUR_USERNAME`).
+3. Set **Password** to the secret you created (i.e. `secrets.REST_BASIC_AUTH_SECRET`).
 
 ![rest Connector basic auth](../img/connectors-rest-basic-auth.png)
```
Comment (MoAtCamunda) [inline docs/components/connectors/out-of-the-box-connectors/rest.md:49]: Should there be a secrets.YOUR_USERNAME here? if not should we use <YOUR_USERNAME>?

**Review by MoAtCamunda — CHANGES_REQUESTED** [review]

**Comment by igpetrov** [comment]
@Oleksiivanov the `build-docs` is failing. You'll likely need to rebase.

**Review by igpetrov — APPROVED** [review]

**Comment by igpetrov** [comment]
Hi @christinaausley,

This one is ready, can we merge it please. The related back-end code is completed and merged.

**Pull request merged**